### ISSUE TITLE
Can't handle texture for TransparencyFactor; discarding.

### ISSUE THREAD

**Issue by southkorea2013** [facebookincubator/FBX2glTF#158]
Hello,

When converting a fbx file to gltf, it always complain that "Can't handle texture for TransparencyFactor; discarding.", so I will miss the opacity image from the original models.
Please shed some light on it. Thanks.
[fbx.zip](https://github.com/facebookincubator/FBX2glTF/files/2805931/fbx.zip)

**Comment by zellski** [comment]
Hi @southkorea2013; thank you very much for this bug report & the test file! This is the first example asset I've received that actually uses a TransparencyFactor texture. I may be able to do something with it –– but it might be a while before I have a chance to pick up the code again.

**Comment by xuejianbin** [comment]
Is this problem solved?

**Comment by zellski** [comment]
@xuejianbin No. I don't foresee having time to work on it anytime soon. If you or someone you know can help, pull requests are always welcome.

**Comment by ehs035** [comment]
Any news on this? I really need this to be fixed. Thanks.

**Comment by zellski** [comment]
@ehs035 do you have a test FBX you can post that includes a transparency map?

**Comment by ehs035** [comment]
> @ehs035 do you have a test FBX you can post that includes a transparency map?

Can I send you PM with the link? Model is pretty big.

**Comment by ehs035** [comment]
> > @ehs035 do you have a test FBX you can post that includes a transparency map?
> 
> Can I send you PM with the link? Model is pretty big.
@zellski  Here is a sample its pretty big: [link](http://www.arbinternational.net/private/tammy.zip)

**Comment by zellski** [comment]
Oops, missed this. Of course you can always email (zell@fb.com) or DM me on Twitter (@zellski). But I'm downloading the link now... veeery... slowly. :-)

**Comment by ehs035** [comment]
> Oops, missed this. Of course you can always email ([zell@fb.com](mailto:zell@fb.com)) or DM me on Twitter (@zellski). But I'm downloading the link now... veeery... slowly. :-)

Sorry wasn't that bad for me, like 3 mins. I should bring a USB drive last week when I was at FB HQ in Menlo Park :-)

**Comment by zellski** [comment]
Hah! Say hi next time; I'm in the MPK office.

**Comment by looeee** [comment]
Hey guys! Just dropping by from the three.js repo. 

We encountered a file with TransparencyFactor here:

https://github.com/mrdoob/three.js/pull/17279

It's the first time in several years working with FBX files that I've seen this map. It seems to be an alpha cutout map, while the one from the tree and swings linked above looks like a standard transparency map. 

I'm going to treat this the same way as `TransparentColor` for now and see if that causes problems.

**Comment by Jarred-Sumner** [comment]
If you want another example, the eyelash material in this character asset has a TransparencyFactor:  https://assetstore.unity.com/packages/3d/characters/o3n-uma-race-stunner-jane-175172

This is what the eyelash renders like: (in Babylon.js)
![image](https://user-images.githubusercontent.com/709451/87863916-f5968500-c915-11ea-807f-1f24e7861faa.png)

Warning you that it shows NSFW content if you convert the .fbx file to .gltf

**Comment by se7enXF** [comment]
Meet the same issue. Anyone has a good solution to make material transparent work?
